azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -6,28 +6,37 @@
 import time
 import uuid
 from typing import Union
+from urllib.parse import urlparse
 
 from azure.kusto.data import KustoClient, KustoConnectionStringBuilder
+from azure.kusto.data.exceptions import KustoServiceError
 from azure.storage.blob import BlobServiceClient
 from azure.storage.queue import QueueServiceClient, TextBase64EncodePolicy
 
 from ._ingestion_blob_info import _IngestionBlobInfo
 from ._resource_manager import _ResourceManager
 from .descriptors import BlobDescriptor, FileDescriptor
+from .exceptions import KustoInvalidEndpointError
 from .ingestion_properties import DataFormat, IngestionProperties
 
 
 class KustoIngestClient:
     """
-    Kusto ingest client provides methods to allow ingestion into kusto (ADX).
+    Kusto queued ingest client provides methods to allow ingestion into kusto (ADX).
```

Comment:
```suggestion
    Kusto ingest client provides methods to allow queued ingestion into kusto (ADX).
```

Reply:
I intentionally left this particular one as is because it's more accurate regardless of what the class is called, but I also like how you wrote it.